_pages/fsa/dynamicforms/styling.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Dynamicforms Styling
+---
+
+Dynamic forms Library uses Bootstrap 4 classes for styling. **Custom** CSS classes can be added in four places while defining JSON form definition through Back-Office.
+
+Custom CSS class can be defined for:
+
+- Form tag - By default empty. This class can be used for specific styling of each form. It can be added by defining **"cssClass"** property in JSON.
+
+Example:
+```typescript
+{
+  "cssClass": "customFormClass", // This can be used to add custom styling class on the form tag
+  "formGroups": [
+    {
+      "groupCode": "firstFormSection",
+      "fieldConfigs": [
+          // controls defined here...
+```
+
+- Form group (section) - By default Dynamicforms will render "row" Bootstrap class here. Defining cssClass in "formGroups" section of JSON will **add** class to default class. It can be added by defining **"cssClass"** property in JSON.
+
+Example:
+```typescript
+{
+  "cssClass": "customFormClass",
+  "formGroups": [
+    {
+      "groupCode": "firstFormSection",
+       "cssClass": "customSectionClass", // This can be used to add custom styling class to the section (Form Group),
```

Comment:
Remove space before "cssClass"

Reply:
> And also comment maybe above the property to make it more readable, because user needs to scroll to the right to reach end of text
This depends on the size of the comment. Even if we move them above the property some are still long enough to be scroll-able. In my opinion, with this implementation user shouldn't be in any doubt what comment is referencing what property.